Source/Header/DayHeaderView.swift

```diff
@@ -129,11 +133,19 @@ public final class DayHeaderView: UIView, DaySelectorDelegate, DayViewStateUpdat
     if daysFrom < 0 {
       currentWeekdayIndex = abs(daysInWeek + daysFrom % daysInWeek) % daysInWeek
       new.selectedIndex = currentWeekdayIndex
-      pagingViewController.setViewControllers([new], direction: .reverse, animated: true, completion: nil)
+        
+      let ltr = UIView.userInterfaceLayoutDirection(for: self.semanticContentAttribute) == .leftToRight
```

Comment:
You could move the `ltr` (also renamed to `leftToRight `) up to have a wider scope and use that variable to choose the direction in both path of the flow.